content/en/style-guide/_index.md

```diff
@@ -79,19 +88,31 @@ title: Definition Template
 status: Feedback Appreciated
 ```
 
-### Category
+### Tags
+
+The **tag** label follows the status label. 
+For the tags to be meaningful and thus helpful to the user, we will use them in a strict sense. 
+Applying too many tags will only delute its meaning. 
+With the exception of "fundamental," which simply indicates this term is needed to understand other cloud native concepts, most terms will likely only have one tag.
+
+**Note**: Please do not introduce new tags unless approved by the maintainers. When you add tags to an entry, ensure they are spelled exactly as listed below (singular, no typos).
 
-The **category** label will come after the status label. Its value should be one of the following values:
+The current tags are:
 
-- Technology
-- Property
-- Concept
+- application
+- architecture
+- fundamental
+- infrastructure
+- methodology
+- networking
+- property
+- security
 
 ```md
 ---
 title: Definition Template
 status: Feedback Appreciated
-category: Concept
```

Comment:
Currently each markdown file for terms still has the 'category' front-matter.
It would be great to remove those (in another PR) if needed. 😊